src/pages/[slug]/[ie]/index.tsx

```diff
@@ -262,6 +260,12 @@ const ObjectDetailPage: NextPage = () => {
 		}
 	};
 
+	const onExportClick = async () => {
+		const xmlBlob = await getMediaExport(router.query.ie as string);
+
+		xmlBlob && save(xmlBlob, `${kebabCase(mediaInfo?.name) || 'metadata'}.xml`);
```

Comment:
Als we de lege response niet opvangen kan het zijn dat de gebruiker op de export knop klikt en er niets gebeurt
```suggestion
		    if (xmlBlob) {
		    	save(xmlBlob, `${kebabCase(mediaInfo?.name) || 'metadata'}.xml`);
		    } else {
			      toastService.notify({
				        title: i18n?.t('error') || 'error',
				        description: i18n?.t('De server gaf een leeg antwoord') || 'error',
			      });
		    }
```